src/playlist-loader.js

```diff
@@ -227,6 +228,44 @@ const PlaylistLoader = function(srcUrl, hls, withCredentials) {
     }
   };
 
+  /**
+   * Returns the number of enabled playlists on the master playlist object
+   *
+   * @return {Number} number of eneabled playlists
+   */
+  loader.enabledPlaylists = function() {
+    return loader.master.playlists.filter((element, index, array) => {
+      return element.excludeUntil <= Date.now() || !element.excludeUntil ? true : false;
+    }).length;
+  };
+
+  /**
+   * Returns whether the current playlist is the lowest rendition
+   *
+   * @return {Boolean} true if on lowest rendition
+   */
+  loader.onLowestEnabledRendition = function() {
+    if (!loader.media()) {
+      return false;
+    }
+
+    let currentPlaylist = loader.media().attributes.BANDWIDTH;
+
+    return loader.master.playlists.filter((element, index, array) => {
+      let enabled = typeof element.excludeUntil === 'undefined' ||
```

Comment:
Originally the check was to call `enabledPlaylists()` to determine if there was only one enabled playlist left and to call `onLowestRendition()` to check if the lowest rendition was active. This would fail if all but one rendition were disabled, but the enabled one was not the lowest. Since I'm not too familiar with how the player selects a playlist, this may not be a use case we have to check for? By adding the check for being enabled in this filter, it fixes that problem and removes the need to have to call `enabledPlaylists()` in the check at all. I left `enabledPlaylists()` in since I was under the impression the method was desired. If that is not the case we can remove it.